src/main/java/cz/muni/fi/pa165/kodemon/entity/Stadium.java

```diff
@@ -1,7 +1,96 @@
 package cz.muni.fi.pa165.kodemon.entity;
 
+import cz.muni.fi.pa165.kodemon.enums.PokemonType;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
 /**
- * Created by mseleng on 10/19/16.
+ * Entity that represents a pokemon stadium.
+ *
+ * Each stadium has a leader, which can be challenged by other trainers that want to earn a badge.
+ *
+ * @author Michal Romanek
  */
+
+@Entity
 public class Stadium {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @NotNull
+    private String city;
+
```

Comment:
Pri `enum` fieldoch musi byt anotacia `@Enumerable`. To som si nevsimol, ci bolo u Pokemona, idem ceknut.